todo app part 1

masha-girya opened the pull request:
[DEMO LINK](https://masha-girya.github.io/react_todo-app-loading-todos/)

viktoriia-ovcharenko commented on src/App.tsx:
```diff
@@ -37,187 +78,39 @@ export const App: React.FC = () => {
           </form>
```
on the mockup, this button is visible when we have some todos

viktoriia-ovcharenko reviewed (approved):
good job!

masha-girya commented:
I was already working on the second part of this task, so there I:
1. added class to Toggle button as Victoria said
2. created NewTodo file where it is adding todo form (saw this in the checklist)
3. changed todos filter (see in comments)

masha-girya commented on src/App.tsx:
```diff
@@ -14,6 +27,34 @@ export const App: React.FC = () => {
+    setError('');
+  };
+
+  const visibleTodos = todos.filter(todo => {
+    switch (selectedFilter) {
+      case Filter.ALL:
+        return todo;
+
+      case Filter.ACTIVE:
+        return !todo.completed;
+
+      case Filter.COMPLETED:
+        return todo.completed;
+
+      default:
+        return todo;
+    }
+  });
+
```
```suggestion
  const filterTodos = (filterBy: Filter) => {
    return (
      todos.filter(todo => {
        switch (filterBy) {
          case Filter.ALL:
            return todo;

          case Filter.ACTIVE:
            return !todo.completed;

          case Filter.COMPLETED:
            return todo.completed;

          default:
            return todo;
        }
      }));
  };

  const filteredTodos = filterTodos(filter);

  const activeTodos = todos.filter(todo => !todo.completed);
```

Olexandr6 reviewed (approved):
Well done 🔥
